cmd/seed.go

```diff
@@ -0,0 +1,36 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// seedCmd represents the seed command
+var seedCmd = &cobra.Command{
+	Use:   "seed",
+	Short: "Share part or all of the index via IPFS.",
+	Long: `A longer description that spans multiple lines and likely contains examples
```

Comment:
Remove boilerplate here as well (or fill in!)

Reply:
I'm going to leave this in for now. I'm not even sure these addition commands (seed, leech, etc.) will remain. We can remove them as we work on these other options.